config/config-api/src/main/java/com/thoughtworks/go/config/ValidationContext.java

```diff
@@ -84,5 +88,9 @@ default SystemEnvironment systemEnvironment() {
     default PolicyValidationContext getPolicyValidationContext() {
         return null;
     }
+
+    default Map<CaseInsensitiveString, Boolean> getPipelinesWithMaterial(String fingerprint) {
```

Comment:
Same as above

Reply:
Couldn't think of a better name. Any suggestions?

Reply:
Updated to `getPipelineToMaterialAutoUpdateMapByFingerprint`